config/breadcrumbs.rb

```diff
@@ -35,6 +35,16 @@
   parent :editorial_section, node.section
 end
 
+crumb :public_news_articles do |section|
+  link 'News', news_articles_path(section.slug)
+  parent :public_node, section.home_node
+end
+
+crumb :public_news_article do |node|
+  link node.name, news_article_path(node.section.home_node.slug, node.slug)
+  parent :public_news_articles, node.section
+end
+
```

Comment:
If I'm being pedantic, these breadcrumbs should live with the other public breadcrumbs rather than in the middle of the editorial ones.